input.go

```diff
@@ -82,10 +82,10 @@ type CategoryUpdateInput struct {
 // CheckAlertSourceUsageCreateInput Specifies the input fields used to create an alert source usage check
 type CheckAlertSourceUsageCreateInput struct {
 	AlertSourceNamePredicate *PredicateInput         `json:"alertSourceNamePredicate,omitempty" yaml:"alertSourceNamePredicate,omitempty"`           // The condition that the alert source name should satisfy to be evaluated (Optional)
-	AlertSourceType          *AlertSourceTypeEnum    `json:"alertSourceType,omitempty" yaml:"alertSourceType,omitempty" example:"datadog"`           // The type of the alert source (Optional)
+	AlertSourceType          *AlertSourceTypeEnum    `json:"alertSourceType,omitempty" yaml:"alertSourceType,omitempty" example:"custom"`            // The type of the alert source (Optional)
```

Comment:
This is changing because the enum's where changed last time to have a new value in the `[0]` spot of the array so the code gen is updating the example to the `[0]` value in the array.